selfservice/strategy/code/strategy_login_test.go

```diff
@@ -451,6 +451,48 @@ func TestLoginCodeStrategy(t *testing.T) {
 				require.NotNil(t, va)
 				require.True(t, va.Verified)
 			})
+
+			t.Run("case=should not populate on 2FA request", func(t *testing.T) {
+				ctx := context.Background()
+
+				// enable webauthn 2FA method
+				conf.MustSet(ctx, fmt.Sprintf("%s.%s.enabled", config.ViperKeySelfServiceStrategyConfig, "webauthn"), true)
```

Comment:
```suggestion
				conf.MustSet(ctx, strings.Join([]string{config.ViperKeySelfServiceStrategyConfig, "webauthn", "enabled"}, "."), true)
```

Reply:
seems awfully complicated for a test all of a sudden :sweat_smile:

Reply:
I guess a bit easier is
```suggestion
				conf.MustSet(ctx, config.ViperKeySelfServiceStrategyConfig + ".webauthn.enabled"), true)
```